packages/aws-cdk-lib/aws-dynamodb/lib/shared.ts

```diff
@@ -218,6 +218,23 @@ export enum StreamViewType {
   KEYS_ONLY = 'KEYS_ONLY',
 }
 
+/**
+ * The precision associated with the DynamoDB write timestamps that will be replicated to Kinesis.
+ * The default setting for record timestamp precision is microseconds. You can change this setting at any time.
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-dynamodb-table-kinesisstreamspecification.html#aws-properties-dynamodb-table-kinesisstreamspecification-properties
+ */
+export enum ApproximateCreationDateTimePrecision {
```

Comment:
I believe this was added in the other PR https://github.com/aws/aws-cdk/pull/31863. Since this also uses it, let's hold off on this PR and remove the definition from https://github.com/aws/aws-cdk/pull/31863 and use the shared enum.

Reply:
Good idea, i'll wait for #31863 to be merged and then i'll update this one.

Reply:
@LeeroyHannigan FYI the other PR is merged now.

Reply:
@GavinZZ Thanks, updated and ready to review.